models/staging/edfi_3/stage/stg_ef3__credentials.sql

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
@holdorph @zaharenka  sometimes we have an empty line at the beginning of the file, sometimes not, maybe it makes sense to do it uniformly?

Reply:
Where it's noticed, I would remove it.  All of the files from the original repository (not created by us) do NOT start with a blank line.